db/seeds.rb

```diff
@@ -17,6 +17,9 @@
   {name: 'Plymouth County House of Correction'}
   ])
 
+Court.create([
```

Comment:
Do you want to reach out to Atara and ask her for a list of these? You could also generate a starting list by looking at the data we currently have in the court field on clients and getting a list of distinct entries.

It probably won't be complete right off the bat, but we'd want it to have as much data as possible before merging this since we'd be going from a free text field to a dropdown